enterprise/models.py

```diff
@@ -4082,7 +4082,8 @@ def submit_for_configuration(self, updating_existing_record=False):
         config_data = {}
         if self.identity_provider == self.SAP_SUCCESS_FACTORS:
             for field in self.sap_config_fields:
-                sap_data[utils.camelCase(field)] = getattr(self, field)
+                if getattr(self, field):
+                    sap_data[utils.camelCase(field)] = getattr(self, field)
```

Comment:
Don't need to call getattr twice
```suggestion
                if hasattr(self, field):
                    sap_data[utils.camelCase(field)] = getattr(self, field)
```
or
```suggestion
                if field_value := getattr(self, field):
                    sap_data[utils.camelCase(field)] = field_value
```